project.clj

```diff
@@ -50,7 +50,10 @@
                  [akiroz.re-frame/storage "0.1.2"]
                  [cljsjs/chartjs "2.6.0-0"]
                  [org.web3j/core "2.3.1"]
-                 [cljs-web3 "0.19.0-0-7"]]
+                 [cljs-web3 "0.19.0-0-7"]
+                 [reifyhealth/specmonstah "1.0.0"]
+                 [clojure-future-spec "1.9.0-beta4"]
```

Comment:
We're on 1.9.0 now so we probably no longer need this dependency.

Reply:
nice catch! thanks